doc/source/whatsnew/v0.25.0.rst

```diff
@@ -1133,7 +1133,7 @@ Groupby/resample/rolling
 - Bug in :meth:`pandas.core.window.Rolling.median` and :meth:`pandas.core.window.Rolling.quantile` where MemoryError is raised with empty window (:issue:`26005`)
 - Bug in :meth:`pandas.core.window.Rolling.median` and :meth:`pandas.core.window.Rolling.quantile` where incorrect results are returned with ``closed='left'`` and ``closed='neither'`` (:issue:`26005`)
 - Improved :class:`pandas.core.window.Rolling`, :class:`pandas.core.window.Window` and :class:`pandas.core.window.EWM` functions to exclude nuisance columns from results instead of raising errors and raise a ``DataError`` only if all columns are nuisance (:issue:`12537`)
-- Bug in :meth:`pandas.core.window.Rolling.max` and :meth:`pandas.core.window.Rolling.min` where incorrect results are returned with an empty variable window`` (:issue:`26005`)
```

Comment:
Should “window” be ticked?

Reply:
Yeah, I wasn't sure if the intention was to tick "window" or not.  The other instances of "window" in the whatsnew are not ticked, so I opted to leave this one unticked for consistency, but can tick it if that's be best course of action.

Reply:
Doesn't need to be ticked - just referring to the concept of a window not an object